### PR TITLE
[ci skip] Add Issue Forms

### PR DESCRIPTION
**Pull request by Nacioszeczek**
This adds issue forms.
See [here](https://github.com/Nacioszeczek/Velocity/issues/new/choose) for preview.

### CODE REVIEW

**Comment by Timongcraft** [comment]
Maybe add title prefixes for bug reports and feature requests so they can be easily identified.

**Comment by Nacioszeczek** [comment]
> Maybe add title prefixes for bug reports and feature requests so they can be easily identified.

This is what labels are for

**Comment by Timongcraft** [comment]
Makes sense, right.

**Review by 4drian3d — APPROVED** [review]

**Review by powercasgamer — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      description: |
+        Please describe as best as you can what you'd like to be added to Velocity.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Why is this needed?
+      description: |
+        Please describe why do you need this feature.
```
Comment (powercasgamer) [inline .github/ISSUE_TEMPLATE/feature-request.yml:17]:
```suggestion
        Please describe why you need this feature.
```

**Review by powercasgamer — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 1 above)
Comment (powercasgamer) [reply .github/ISSUE_TEMPLATE/feature-request.yml:17]: NOT RESOLVED!

**Pull request merged**